nemo/collections/common/data/lhotse/dataloader.py

```diff
@@ -438,3 +444,20 @@ def _flatten_alt_text(cut) -> list:
         text_instance.custom = {"text": data.pop("text"), "lang": data.pop("lang"), **data}
         ans.append(text_instance)
     return ans
+
+
+def _select_channel(cut, channel_selector: int | str) -> list:
+    if isinstance(channel_selector, int):
+        channel_idx = channel_selector
+    elif isinstance(channel_selector, str):
+        if channel_selector in cut.custom:
+            channel_idx = cut.custom[channel_selector]
+        else:
+            raise ValueError(f"Channel selector {channel_selector} not found in cut.custom")
+
+    if channel_idx >= cut.num_channels:
+        raise ValueError(
+            f"Channel index {channel_idx} is larger than the actual number of channels {cut.num_channels}"
+        )
+
+    return cut.with_channels(channel_idx)
```

Comment:
hmmm `.with_channels` is only defined on `MultiCut`, perhaps we should add a check like:
```
if cut.num_channels == 1: 
    return cut
else:
    return cut.with_channels(channel_idx)
```

WDYT?